docs/monitoring.md

```diff
@@ -25,6 +25,27 @@ To view the web UI after the fact, set `spark.eventLog.enabled` to true before s
 application. This configures Spark to log Spark events that encode the information displayed
 in the UI to persisted storage.
 
+### Spark Web UI Configuration Options
```

Comment:
These are in `configuration.md`.

Reply:
Ah thanks for pointing it out! Will move.